packages/core/src/resolver.js

```diff
@@ -46,7 +46,7 @@ export default class Resolver {
     return resolve(this.options.srcDir, path)
   }
 
-  resolvePath(path, { alias, module } = {}) {
+  resolvePath(path, { alias, module, isStyle } = {}) {
```

Comment:
Maybe we should move to `isXYZ` syntax generally for booleans here? :thinking:

Reply:
I like this convention :+1: